.github/workflows/selfdrive_tests.yaml

```diff
@@ -289,9 +289,9 @@ jobs:
           ref: ${{ github.event.pull_request.base.ref }}
```

Comment:
We want to get rid of the two checkouts in this PR and just use curl/wget

Reply:
Done (https://github.com/commaai/openpilot/pull/33273/commits/ddbe96471e627abb5fe22f0e8df54cd1a9e25260)